web/styles/app_variables.css

```
@@ -40,7 +40,15 @@
     );
     /* Keep content from colliding with the search box. */
     --navbar-content-righthand-offset: 5px;
-
+    /* 40px (Sidebar toggle icon) + 80px (wide logo) */
```

Will be good to add a blank line above and change the comment to the given suggestion. 
```suggestion

   /* Width of navbar left column
   40px (Sidebar toggle icon) + 80px (wide logo). */
```

---

Done!